src/main/java/vg/civcraft/mc/civmodcore/api/NBTCompound.java

```diff
@@ -87,6 +87,15 @@ public void clear() {
 		}
 	}
 
+	/**
+	 * Clones the NBT compound.
+	 *
+	 * @return Returns a duplicated version of this NBT compound.
+	 * */
+	public NBTCompound duplicate() {
```

Comment:
Java convention would be to implement https://docs.oracle.com/javase/7/docs/api/java/lang/Cloneable.html for this functionality

Reply:
Done